draft-ietf-httpbis-digest-headers.md

```diff
@@ -586,16 +586,19 @@ Digest: sha-256=X48E9qOokqqrvdts8nOJRJN3OWDUoyWxBf7kbu9DBPE=
 
 ## Server Returns No Representation Data
 
+In this example, a HEAD request is used to retrieve the checksum
+of a resource.
```

Comment:
I'd move this to after the next paragraph (e.g. after L596 as it presently is in the PR).

Reply:
:heavy_check_mark: